src/components/CandidateCategoryTile.jsx

```diff
@@ -33,7 +37,7 @@ const CandidateCategoryTile = ({ slugs, category }) => {
             name={t(`category.${category}`)}
             IconContent={
               <Grid container spacing={0}>
-                {slugs.map(slug => (
+                {slugs.slice(0, 4).map(slug => (
```

Comment:
Why this change? I don't understand it from the commit message?

Reply:
It is because it is possible now to have more than 4 konnectors here (because I added more in candidates.json).

Reply:
> It is because it is possible now to have more than 4 konnectors here (because I added more in candidates.json)

We should rely on `MIN_SLUGS` then?